src/main/java/uk/gov/hmcts/sptribs/caseworker/model/CaseFlag.java

```diff
@@ -64,4 +68,78 @@ public class CaseFlag {
     )
     private List<ListValue<Flags>> caseFlags;
 
+    @CCD(
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private List<ListValue<CaseFlagDisplay>> partyLevelFlags;
+
+    @CCD(
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private List<ListValue<CaseFlagDisplay>> caseLevelFlags;
+
+    @SuppressWarnings("unchecked")
+    public List<ListValue<CaseFlagDisplay>> getPartyLevelFlags() {
```

Comment:
Do we need SuppressWarnings("unchecked") here? Is it possible to code the generics so as to be able to remove this?

Reply:
Could not find a way to work without them

Reply:
I checked out the branch to take a look. The first 2 methods aren't used, so it doesn't really check whether the suppression is needed or not, and the 3rd suppression is apparently redundant, so can be removed?